### PR TITLE
move MAV_CMD_FIXED_MAG_CAL_YAW ardupilotmega -> common

### PR DESCRIPTION
**Pull request by dagar**
https://github.com/mavlink/qgroundcontrol/issues/8142

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
Looks fine by me. I assume it's ok to move straight across. @auturgy Do you know of any issues doing this?

**Comment by DonLakeFlyer** [comment]
Then once this gets merged I'll do this: https://github.com/mavlink/qgroundcontrol/issues/8142

File: message_definitions/v1.0/ardupilotmega.xml [hunk 1]
```diff
@@ -112,16 +112,7 @@
...
-        <description>Magnetometer calibration based on provided known yaw. This allows for fast calibration using WMM field tables in the vehicle, given only the known yaw of the vehicle. If Latitude and longitude are both zero then use the current vehicle location.</description>
-        <param index="1" label="Yaw" units="deg">Yaw of vehicle in earth frame.</param>
-        <param index="2" label="CompassMask">CompassMask, 0 for all.</param>
-        <param index="3" label="Latitude" units="deg">Latitude.</param>
-        <param index="4" label="Longitude" units="deg">Longitude.</param>
-        <param index="5">Empty.</param>
-        <param index="6">Empty.</param>
-        <param index="7">Empty.</param>
-      </entry>
+      <!-- 42006 used by common.xml -->
```
Comment (auturgy) [inline message_definitions/v1.0/ardupilotmega.xml:115]: I'd prefer to have the message name in the comment here, for traceability.
ie

**Review by auturgy — APPROVED** [review]
Minor change requested for the comment in ardupilotmega.

**Comment by auturgy** [comment]
@DonLakeFlyer 
> Looks fine by me. I assume it's ok to move straight across. @auturgy Do you know of any issues doing this?

No, I much prefer this approach than having functionally duplicated messages.  Aids moving towards greater compatibility.

File: message_definitions/v1.0/ardupilotmega.xml (hunk 1 above)
Comment (dagar) [reply message_definitions/v1.0/ardupilotmega.xml:115]: Done.

**Comment by dagar** [comment]
Thanks everyone.

**Pull request merged**